app/policies/participant_profile/ecf_policy.rb

```diff
@@ -45,6 +45,13 @@ def withdraw_record?
 private
 
   def same_school?
-    user.induction_coordinator_profile.schools.include?(record.school)
+    if FeatureFlag.active?(:change_of_circumstances)
+      InductionRecord.joins(:school)
```

Comment:
out of curiosity, would this work `InductionRecord.for_school(record.school).where(participant_profile_id: record.id).any?`